sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Models
 {
     internal partial class TelemetryItem
     {
-        public TelemetryItem(Activity activity, ref ActivityTagsProcessor activityTagsProcessor, AzureMonitorResource? resource, string instrumentationKey) :
+        public TelemetryItem(Activity activity, ref ActivityTagsProcessor activityTagsProcessor, AzureMonitorResource? resource, string instrumentationKey, float? sampleRate = null) :
```

Comment:
why sampleRate is passed as nullable?

Reply:
it's nullable in the Model.
https://github.com/Azure/azure-sdk-for-net/blob/bf5d57051e75c36b02a2bc6a948e03de5686f93b/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Generated/Models/TelemetryItem.cs#L36-L37

Reply:
This will never be null, as in the calling method it is declared as non-nullable `private readonly float _sampleRate;`